Release v2.0/voorbeelden maatregelen (#503)
MinBZK/Algoritmekader: docs/voldoen-aan-wetten-en-regels/maatregelen/3-dat-05-pseudonimiseren-anonimiseren.md

- [Toetsingskader Algoritmes Algemene Rekenkamer, 2.20](https://www.rekenkamer.nl/onderwerpen/algoritmes/documenten/publicaties/2024/05/15/het-toetsingskader-aan-de-slag)
## Voorbeeld

!!! example "Groep Gegevensbescherming: Artikel 20 (WP 216)"

    _Dit voorbeeld kan ter inspiratie dienen voor hoe de maatregel toegepast kan worden._

>     In een advies stuk over anonimiseringstechnieken staan verschillende technieken benoemd voor het pseudonimiseren en anonimiseren van persoonsgegevens. Hierbij wordt gekeken naar de factoren: Herleidbaarheid, Koppelbaarheid en Deduceerbaarheid. Hierbij worden ook vaak gemaakte fouten aangegeven zodat deze voorkomen kunnen worden.

Reviewer: ```suggestion
    In een advies van de 'Artikel 29 Werkgroep' (tegenwoordig vervangen door de [European Data Protection Board](https://www.edpb.europa.eu/about-edpb/who-we-are/legacy-art-29-working-party_nl)) over anonimiseringstechnieken staan verschillende technieken benoemd voor het pseudonimiseren en anonimiseren van persoonsgegevens. Hierbij wordt gekeken naar de factoren: Herleidbaarheid, Koppelbaarheid en Deduceerbaarheid. Hierbij worden ook vaak gemaakte fouten aangegeven zodat deze voorkomen kunnen worden.
```

>     Er staat niet exact in hoe deze technieken uit gevoerd moeten worden in dit document. Er wordt hier wel bij iedere techniek een voorbeeld gegeven, maar de exacte implementatie moet zelf verder bekeken worden. Dit voorbeeld kan ondersteunen bij het kiezen van een vorm van gegevens bescherming.
>     _Volgens Autoriteit Persoonsgegevens werkt de European Data Protection Board (EDPB) aan guidelines over anonimiseren en pseudonimiseren, zodra hier meer over bekend is kan deze toegevoegd ook worden._

Reviewer: ```suggestion
    Er wordt bij iedere techniek een voorbeeld gegeven, maar de exacte implementatie moet zelf verder bekeken worden. Dit voorbeeld kan ondersteunen bij het kiezen van een vorm van gegevensbescherming.
    _Volgens de Autoriteit Persoonsgegevens werkt de EDPB aan guidelines over anonimiseren en pseudonimiseren. Zodra hier meer over bekend is, zal dit toegevoegd worden._
```

>     Er wordt bij iedere techniek een voorbeeld gegeven, maar de exacte implementatie moet zelf verder bekeken worden. Dit voorbeeld kan ondersteunen bij het kiezen van een vorm van gegevensbescherming.
>     _Volgens de Autoriteit Persoonsgegevens werkt de EDPB aan guidelines over anonimiseren en pseudonimiseren. Zodra hier meer over bekend is, zal dit toegevoegd worden._

Reviewer: ```suggestion
    Er wordt bij iedere techniek een voorbeeld gegeven, maar de exacte implementatie moet zelf verder bekeken worden. Dit voorbeeld kan ondersteunen bij het kiezen van een vorm van gegevensbescherming.
    _[Volgens de Autoriteit Persoonsgegevens](https://www.autoriteitpersoonsgegevens.nl/themas/beveiliging/beveiliging-van-persoonsgegevens/gegevens-pseudonimiseren#:~:text=De%20European%20Data%20Protection%20Board,privacytoezichthouders%20uit%20de%20EER%20samenwerken.) werkt de EDPB aan guidelines over anonimiseren en pseudonimiseren. Zodra hier meer over bekend is, zal dit toegevoegd worden._
```
